ansible/roles/automation/nightly_pr/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+- name: schedule opening nightly PRs
+  cron:
+    name: "openclose_{{ item.name }}"
+    minute: "{{ item.minute }}"
+    hour: "{{ item.hour }}"
+    weekday: "{{ item.weekdays }}"
+    job: >
+           python3 {{ basedir }}/open_close_pr.py
+           open_nightly_pr
+           --config {{ basedir }}/config.yml
+           --id {{ item.name }}
+           --repo_path {{ basedir }}/freeipa
+           --branch {{ branch }}
```

Comment:
we need to add flow variable here to distinguish between e.g. ci-master-f29 and pki-master-f29.

Reply:
Hy Triviño,
Today, in crontab the line stand like this:
```
0 10 * * 6 python3 /root/openclose_pr/open_close_pr.py --config /root/openclose_pr/config.yml --id testing_rawhide --repo_path /root/openclose_pr/freeipa --branch master --prci_config ipatests/prci_definitions/nightly_rawhide.yaml --pr_against_upstream True open_pr
```
Please notice that no reference to template is done here, but it has a reference to tests file, which has topology and template definition.

If a diferent template must be used, I strongly recommend to create a new test file ( just copying all content and changing to most approprieate template ).

If you want to add the flow variable here, we need to change the ```open_close_pr.py``` argument parser to parse it.

Reply:
With this commit, we will have two different crontab jobs: one for running nightly regressions (what we have now) and another one for automating template upgrades. I want to add flow variable only for the second one, so we will be able to automate different kind of templates such as:
- ci-master-*
- pki-master-*
- etc..

Reply:
yes, I extended open_close_pr.py to support "flow"var.